test/unit/math/mix/fun/identity_constrain_test.cpp

```diff
@@ -2,22 +2,27 @@
 #include <limits>
 
 template <typename T>
-T g1(const T& x) {
-  return stan::math::identity_constrain(x);
+auto g1(const T& x) {
+  auto x_cons = stan::math::identity_constrain(x);
+  auto x_free = stan::math::identity_free(x_cons);
+  return x_free;
 }
 template <typename T>
-T g2(const T& x) {
-  T lp = 0;
-  return stan::math::identity_constrain(x, lp);
+auto g2(const T& x) {
+  stan::value_type_t<T> lp = 0;
+  auto x_cons = stan::math::identity_constrain(x, lp);
+  auto x_free = stan::math::identity_free(x_cons);
+  return x_free;
 }
 template <typename T>
-T g3(const T& x) {
-  T lp = 0;
-  stan::math::identity_constrain(x, lp);
+auto g3(const T& x) {
+  stan::value_type_t<T> lp = 0;
+  auto x_cons = stan::math::identity_constrain(x, lp);
+  auto x_free = stan::math::identity_free(x_cons);
```

Comment:
You are only testing `identity_constrain` and `identity_free` together. Each should also be tested individually.